base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyFindCLI.java

```diff
@@ -78,6 +78,10 @@ public void createOptions() {
         option.setArgName("realm");
         options.addOption(option);
 
+        option = new Option(null, "ownerName", true, "ownerName: <tokenID>:<userID>");
```

Comment:
Here's the output of `pki kra-key-find`:
```
  Key ID: 0x39f9a475fc569ed63a0cca110a04a1bf
  Algorithm: 1.2.840.113549.1.1.1
  Size: 2048
  Owner: UID=testuser
```
For consistency with the output, I'd suggest we call the option `--owner` instead of `--ownerName`.

Also it looks like there isn't any specific format for the value, so we should not limit it to `<tokenID>:<userID>`. I'd suggest we change the option description to say just `Owner name`.

Reply:
yeah. I added this option just by keeping TPS in thought. It cleared the doubts that there shouldn't be any format for this filter. thanks.